addons/account/wizard/wizard_tax_adjustments.py

```diff
@@ -20,10 +25,11 @@ def _get_default_journal(self):
                                         domain="[('deprecated', '=', False), ('is_off_balance', '=', False)]")
     amount = fields.Monetary(currency_field='company_currency_id', required=True)
     adjustment_type = fields.Selection([('debit', 'Applied on debit journal item'), ('credit', 'Applied on credit journal item')], string="Adjustment Type", required=True)
-    tax_report_line_id = fields.Many2one(string="Report Line", comodel_name='account.tax.report.line', required=True, help="The report line to make an adjustment for.")
+    tax_report_line_id = fields.Many2one(string="Report Line", comodel_name='account.tax.report.line', required=True, help="The report line to make an adjustment for.",
+                                         domain="[('tag_name', '!=', None), ('report_id.country_id', 'in', fiscal_countries)]")
```

Comment:
Don't be afraid to really break this line. VCS will we happy, and people blaming the code as well.
```suggestion
    tax_report_line_id = fields.Many2one(
        string="Report Line", 
        comodel_name='account.tax.report.line', 
        required=True,
        domain="[('tag_name', '!=', None), ('report_id.country_id', 'in', fiscal_country_ids)]",  # or 'in fiscal_country_ids'
        help="The report line to make an adjustment for.",
    )
```